src/model/AirLoopHVACUnitarySystem.cpp

```diff
@@ -2157,6 +2012,147 @@ namespace model {
                       "setSupplyAirFlowRatePerFloorAreaduringHeatingOperation, or setFractionofAutosizedDesignHeatingSupplyAirFlowRate instead.");
   }
 
+  double AirLoopHVACUnitarySystem::maximumCyclingRate() const {
+    DEPRECATED_AT_MSG(3, 7, 0,
+                      "As of EnergyPlus 23.2.0, this property is on the child cooling coil. Use coolingCoil()->maximumCyclingRate() instead.");
+    double result = 2.5;  // former default
+    if (auto c_ = coolingCoil()) {
+      if (auto c_eq_ = c_->optionalCast<CoilCoolingWaterToAirHeatPumpEquationFit>()) {
+        result = c_eq_->maximumCyclingRate();
+      } else if (auto c_vs_ = c_->optionalCast<CoilCoolingWaterToAirHeatPumpVariableSpeedEquationFit>()) {
+        result = c_vs_->maximumCyclingRate();
+      } else if (auto c_dx_ = c_->optionalCast<CoilCoolingDXVariableSpeed>()) {
+        result = c_dx_->maximumCyclingRate();
+      }
+    }
+    return result;
+  }
```

Comment:
Deprecated the former model methods for the unitary systems, trying to set it on the child cooling coil.

**Note**: This is does not modify the "Part Load Fraction Correlation Curve" on the heating coil (which is created during VT using some calculations based on Max Cycling Rate and Heat Pump Time Constant). That would be very weird IMHO. 
I actually considering just making the deprecated method be a no-op and return false